sass/layout/_post.scss

```diff
@@ -2041,5 +2041,5 @@
     position: absolute;
     right: 0;
     top: 0;
-    width: 100%;
+    width: 0;
```

Comment:
Would using `visibility: hidden` make more sense? Or does that have other effects that cause problems here?
https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

Reply:
Yes, I don't trust too much in this kind of things because some browsers doesn't allow you to interact with not-visible elements, but yes, we can try with other approaches, anyway we need to test it in all the browsers (my solution works in firefox and chromium for sure)

Reply:
I'm not opposed to using `width: 0` I just have the same concern that I'm not sure how all browsers will treat that and if it would still be possible to click or see somehow. I'm 0/5